suites/apis/rasIntegrationTest.js

```diff
@@ -115,11 +115,48 @@ Scenario('Use client creds from RAS Test User 1 and auth code to obtain access t
 
   const tokens = JSON.parse(obtainTokensForRASUser1);
 
+  // store refresh token to be used in the next test scenario
+  I.cache.rasUser1RefreshToken = tokens.refresh_token;
+
   // decode JWT / Access Token
   const accessTokenJson = parseJwt(tokens.access_token);
 
-  console.log(`access token scopes: ${accessTokenJson.aud}`);
+  // PXP-6552 - Also decode the id_token and check if the client id is in place
+  const idTokenJson = parseJwt(tokens.id_token);
+  expect(idTokenJson).to.have.property('aud');
+  expect(idTokenJson.aud).to.include(I.cache.rasUser1ClientId);
+
+  // run curl with access token and assert the claim is in place
+  const userInfoOutputCmd = `curl -H "Authorization: bearer ${tokens.access_token}" https://${process.env.HOSTNAME}/user/user`;
+  const userInfoOutput = bash.runCommand(userInfoOutputCmd);
+  console.log(`userInfoOutput: ${JSON.stringify(userInfoOutput.ga4gh_passport_v1)}`);
+  const userInfoOutputJson = JSON.parse(userInfoOutput);
+
+  expect(userInfoOutputJson).to.have.property('ga4gh_passport_v1');
+  expect(userInfoOutputJson.ga4gh_passport_v1).to.not.to.be.empty;
+
+  console.log(`access token audience: ${accessTokenJson.aud}`);
+  console.log(`access token scope: ${accessTokenJson.scope}`);
+  expect(accessTokenJson.aud).to.include('ga4gh_passport_v1');
```

Comment:
oh this you can get rid of as longer term we don't want scopes in `aud` and is not a requirement from ga4gh